docs/conformance-test.md

```diff
@@ -63,13 +63,11 @@ make run
 
```

Comment:
Nit:  can you change
`kubectl apply -f example my-hotel-gateway.yaml` to
`kubectl apply -f examples/my-hotel-gateway.yaml` ?